js/components/multi-audio/multiAudioPopover.js

```diff
@@ -0,0 +1,62 @@
+var React = require('react'),
+  CONSTANTS = require('../../constants/constants'),
+  AccessibleMenu = require('../higher-order/accessibleMenu'),
+  CloseButton = require('../closeButton');
+
+var MultiAudioPopover = React.createClass({
+  /**
+   * close the popover
+   */
+  handleClose: function() {
+    this.props.togglePopoverAction({
+      restoreToggleButtonFocus: true
+    });
+  },
+
+  /**
+   * Call setCurrentAudio from controller
+   * @param {String} id - id of the audio track to activate
+   */
+  handleMultiAudioClick: function(id) {
+    //change audio only if try to set not current audio track
+    if (this.props.controller && this.props.controller.state.currentAudioId !== id) {
+      if (typeof this.props.controller.setCurrentAudio === 'function') {
+        this.props.controller.setCurrentAudio(id);
+      }
+    }
+  },
+
+  render: function() {
+    var multiAudio = [];
+    if (this.props.controller && this.props.controller.state && !!this.props.controller.state.multiAudio) {
+      multiAudio = this.props.controller.state.multiAudio.multiAudio;
```

Comment:
Should `multiAudio.multiAudio` be something like `multiAudio.tracks`? It's not clear what the second `multiAudio` refers to.

Reply:
\+